src/app/game/game-intermediate-result/game-intermediate-result-header/game-intermediate-result-header.component.scss

```diff
@@ -14,3 +14,22 @@
   align-self: center;
   justify-self: center;
 }
+
+.home-btn {

```

Comment:
Could we add some feedback on click? And potentially hover? Can use the next-drawing/summary button for inspiration